test/errors.js

```diff
@@ -9,5 +9,5 @@ test('mergeImages rejects Promise if node-canvas instance isn\'t passed in', asy
 
 test('mergeImages rejects Promise if image load errors', async t => {
 	t.plan(1);
-	await t.throws(mergeImages([1], { Canvas }));
+	await t.throws(mergeImages(['nothing-here.jpg'], { Canvas, Image }));
```

Comment:
How come this is changed to `'nothing-here.jpg'` instead of `1`?

Reply:
Without this it now fails with

```
  errors › mergeImages rejects Promise if image load errors

  /Users/tomjenkinson/Documents/GitHub/merge-images/node_modules/nyc/node_modules/signal-exit/index.js:155

  Promise returned by test never resolved
```

I investigated a bit further and the reason is that `Image` doesn't call `onerror` (or `onload`) when it's given a number. Chrome does call `onerror`, so this probably should be a bug report on `Canvas`